Getting started content: Platform demo (RNASeq) and other updates (#124)
seqeralabs/docs: platform_versioned_docs/version-24.1.1/getting-started/quickstart-demo/data-studios.mdx

Deploy the RShiny app in the data studio by selecting the green play button on the last chunk of the R script:

![Run RShiny app](./assets/rnaseq-diffab-run-rshiny-app.png)

### Explore results in the RShiny app

The RShiny app will deploy in a separate browser window, providing a data interface. Here you can view information about your sample data, perform QC or exploratory analysis, and view the results of differential expression analyses.

![RShiny app exploration](assets/rnaseq-diffab-rshiny-app-explore.gif)

#### Sample clustering with PCA plots

In the **QC/Exploratory** tab, select the PCA (Principal Component Analysis) plot to visualize how the samples group together based on their gene expression profiles.

In this example, we used RNA sequencing data from the publicly-available ENCODE project, which includes samples from four different cell lines: 

- **GM12878** — a lymphoblastoid cell line
- **K562** — a chronic myelogenous leukemia cell line
- **MCF-7** — a breast cancer cell line

> - **H1-hESC** — human embryonic stem cells

Reviewer: I thought H1-hESC was a cell line, the wording is nuanced. 
```suggestion
- **H1-hESC** — a human embryonic stem cell line
```

What to look for in the PCA plot:

> - **Replicate clustering**: Ideally, replicates of the same cell type should cluster closely together. For example, replicates of MCF-7 (breast cancer cell line) group together. This indicates consistent gene expression profiles among replicates.

Reviewer: When writing publications you would refer to replicates as biological or technical replicates. It's not overly important in this context but it shows you are serious about the underlying biology that might influence the clustering.

> - **Cell type separation**: Different cell types should form distinct clusters. For instance, GM12878, K562, MCF-7, and H1-hESC samples should each form their own separate clusters, reflecting their unique gene expression patterns.

Reviewer: ```suggestion
- **Cell type separation**: Different cell types should form distinct clusters. For instance, GM12878, K562, MCF-7, and H1-hESC cells should each form their own separate clusters, reflecting their unique gene expression patterns.
```

From this PCA plot, you can gain insights into the consistency and quality of your sequencing data, identify any potential issues, and understand the major sources of variation among your samples - all directly in Platform.

![RShiny PCA plot](assets/rnaseq-diffab-rshiny-pca-plot.gif)

#### Gene expression changes with Volcano plots 

In the **Differential** tab, select **Volcano plots** to compare genes with significant changes in expression between two samples. For example, filter for `Type: H1 vs MCF-7` to view the differences in expression between these two cell lines.

> 1. **Identify upregulated and downregulated genes**: The x-axis of the volcano plot represents the log2 fold change in gene expression between the H1 and MCF-7 samples, while the y-axis represents the statistical significance of the changes.

Reviewer: ```suggestion
1. **Identify upregulated and downregulated genes**: The x-axis of the volcano plot represents the log2 fold change in gene expression between the H1 and MCF-7 cell lines, while the y-axis represents the statistical significance of the changes.
```

>     - **Upregulated genes in MCF-7**: Genes on the left side of the plot (negative fold change) are upregulated in the MCF-7 samples compared to H1. For example, the SHH gene, which is known to be upregulated in cancer cell lines, prominently appears here.

Reviewer: ```suggestion
    - **Upregulated genes in MCF-7**: Genes on the left side of the plot (negative fold change) are upregulated in the MCF-7 samples compared to H1. For example, the _SHH_ gene, which is known to be upregulated in cancer cell lines, prominently appears here.
```

Reviewer: Genes should be italicised

> 2. **Filtering for specific genes**: If you are interested in specific genes, use the filter function. For example, filter for the SHH gene in the table below the plot. This allows you to quickly locate and examine this gene in more detail.

Reviewer: ```suggestion
2. **Filtering for specific genes**: If you are interested in specific genes, use the filter function. For example, filter for the _SHH_ gene in the table below the plot. This allows you to quickly locate and examine this gene in more detail.
```

> 3. **Gene expression bar plot**: After filtering for the SHH gene, select it to navigate to a gene expression bar plot. This plot will show you the expression levels of SHH across all samples, allowing you to see in which samples it is most highly expressed.

Reviewer: ```suggestion
3. **Gene expression bar plot**: After filtering for the _SHH_ gene, select it to navigate to a gene expression bar plot. This plot will show you the expression levels of _SHH_ across all samples, allowing you to see in which samples it is most highly expressed.
```

>     - Here, SHH is most highly expressed in MCF-7, which aligns with its known role in cancer cell proliferation.

Reviewer: ```suggestion
    - Here, _SHH_ is most highly expressed in MCF-7, which aligns with its known role in cancer cell proliferation.
```
